jenkins-integration/beaker/install/shared/45_install_hiera_config.rb

```diff
@@ -0,0 +1,39 @@
+require 'puppet/gatling/config'
+require 'tmpdir'
+require 'yaml'
+
+test_name 'Install hiera config'
+
+def target_hiera_config_path(host)
+  return on(host, puppet('config print hiera_config')).stdout.chomp
+end
+
+def get_source_hiera_config(host, hiera_config)
+  source_hiera_config_path = hiera_config[:source_file]
+  if source_hiera_config_path.nil? || source_hiera_config_path.empty?
+    source_hiera_config_path = target_hiera_config_path(host)
```

Comment:
I put up #75 to avoid running any of the custom hiera config steps if the job doesn't set them.  I did that at the groovy layer so it avoids touching the hiera configs and service restarting in that case.